include/deal.II/base/vectorization.h

```diff
@@ -180,8 +180,10 @@ class VectorizedArray
    */
   static const unsigned int n_array_elements = 1;
 
-  // POD means that there should be no user-defined constructors, destructors
-  // and copy functions (the standard is somewhat relaxed in C++2011, though).
+  /**
+   * Default constructor
+   */
+  VectorizedArray() = default;
```

Comment:
Don't we still need the constructor
```
DEAL_II_ALWAYS_INLINE explicit VectorizedArray(const Number x)
  : data(x)
{}
```

Reply:
On my machine, if I compile the code, it would never instantiate the general struct `VectorizedArray<Number>`. It always instantiates the specialized versions. So I can't completely test whether we need these constructors or not.

Having said that, yes I think we need to add it.

Reply:
You can try with `Tensor<1,dim,VectorizedArray<unsigned int>>` or `Tensor<1,dim,VectorizedArray<long double>>`, these two do not go into the specialized versions.